README.md

```diff
@@ -120,6 +126,22 @@ cli:
     #    cli-flatten-payload-max-poly-as-resource-prop-count: 8
     #    # max properties allowed from flatten of sub-class as param
     #    cli-flatten-payload-max-poly-as-param-prop-count: 8
+
+    expand-operation:
+        # if true, operation with 'expand-operation-names' will be expanded into multiple 
+        # operation for ecah operation name. 
+        # Notice: expaneded operation's key is in formate: <OriginalOperation>#<ExpandedName>. 
+        # For example, in following case, the expanded operation keys are 'CreateOrUpdate#Create'
+        # and 'CreateOrUpdate#Update'. To make direcitve works on expended operation, please use
+        # the new key
+        cli-expand-operation-enabled: true

```

Comment:
>cli-expand-operation-enabled: true [](start = 8, length = 34)

any case we want to turn it to false?

Reply:
I regard it as a global switch. Refer to `expand-as-resource`